metrics/src/datapoint.rs

```diff
@@ -147,6 +147,13 @@ macro_rules! datapoint_trace {
     };
 }
 
+#[derive(Debug, Clone)]
+pub enum PohTimingPoint {
+    PohSlotStart(u64),
+    PohSlotEnd(u64),
+    FullShredReceived(u64),
```

Comment:
FullSlotReceived maybe. Full Shred doesn't make much sense to me. All shreds are full, they are only one packet.